llvm/test/tools/llvm-objdump/visualize-jumps-aarch64.s

```diff
@@ -0,0 +1,69 @@
+// RUN: llvm-mc < %s -triple aarch64 -filetype=obj | \
+// RUN:   llvm-objdump --triple aarch64 -d --visualize-jumps=unicode - | \
+// RUN:   diff - %p/Inputs/visualize-jumps-aarch64-unicode.txt
```

Comment:
Rather than have the expected files in a separate location, please include them within this file. You can use `split-file` to divide a test file into multiple components (e.g. assembly and expected output).

That being said, a better method of testing would be to use FileCheck, with the expected input being the check patterns used by FileCheck.

Reply:
I used FileCheck for testing similar output with --debug-vars and was never very happy with that, because the disassembly contains tabs, so the output ends up getting misaligned in the text editor. I did it this way so that column 0 of the expected output is at column 0 of the test file, so the test looks exactly like the output.

Thanks for pointing out split-file, I didn't know about that. That looks like the best solution to this, so I'll switch the tests over to using it.

Reply:
> because the disassembly contains tabs, so the output ends up getting misaligned in the text editor. I did it this way so that column 0 of the expected output is at column 0 of the test file, so the test looks exactly like the output.

Assuming you have --strict-whitespace and --match-full-lines on, I'm not sure I understand where this can go wrong, if you're careful with how you line up the colon following the check prefixes? I have a soft preference for FileCheck, as it has better diagnostics when things don't match, and it's easier to omit stuff irrelevant to what is under test.